testing/conformance/src/vector.rs

```diff
@@ -45,9 +45,9 @@ pub struct MetaData {
 pub struct PreConditions {
     pub state_tree: StateTreeVector,
     #[serde(default)]
-    pub basefee: Option<f64>,
+    pub basefee: Option<u128>,
     #[serde(default)]
-    pub circ_supply: Option<f64>,
```

Comment:
@noot FYI -- some of the new vectors might fail with Forest's driver due to this.